### DISCUSSION TITLE
Implement caching for `I18n` gem`

### DISCUSSION THREAD

**Discussion by nimmolo** [MushroomObserver/mushroom-observer#979] [Ideas]
According to the `I18n` gem itself, it's [easy to set up](https://github.com/ruby-i18n/i18n/blob/master/lib/i18n/backend/cache.rb). 

We'd need to have [ActiveSupport::Cache](https://guides.rubyonrails.org/caching_with_rails.html) set up. ("Fragment caching", in Rails, may be on in production by default.) Anyway, it sounds like it could be worth setting up, according to gem authors.

Here's what the gem has to say:
```
# This module allows you to easily cache all responses from the backend - thus
# speeding up the I18n aspects of your application quite a bit.
#
# To enable caching you can simply include the Cache module to the Simple
# backend - or whatever other backend you are using:
#
#   I18n::Backend::Simple.send(:include, I18n::Backend::Cache)
#
# You will also need to set a cache store implementation that you want to use:
#
#   I18n.cache_store = ActiveSupport::Cache.lookup_store(:memory_store)
# ...continued...
```
...where would these lines go? In `config/application.rb`?
(The other [reference](https://lingohub.com/blog/2013/08/i18n-gem-advanced-features-ruby-rails-internationalization) I found is equally vague about where the config goes.)

**Comment by pellaea** [comment]
I would guess config/application.rb, too.  Have you tried it to see if it
works as expected?

**Reply by nimmolo** [reply]
Thanks @pellaea. I haven't, will research what to expect (other than no errors!) and try that next.
